### ISSUE TITLE
Date generated in Changelog doesn't reflect local timezone

### ISSUE THREAD

**Issue by medikoo** [conventional-changelog/conventional-changelog#594]
I'm currently in New Zealand where it's morning of 14th of Feb, while generated changelog was generated with `(2020-02-13)` date header  (I rely on `standard-version` v7.1.0 to generate changelog).

Running `console.log(new Date().toString())` in Node.js produces:

```
Fri Feb 14 2020 09:00:52 GMT+1300 (New Zealand Daylight Time)
```

Therefore it's not an issue with environment setup.

I believe changelog should be generated with date from local timezone (and not GMT or whatever else timezone)

**Comment by tommywo** [comment]
By default time is converted to UTC. If we would use local timezone, it would be possible for two people to make two consecutive releases with first annotated with date that comes after the second release and that would confuse people even more.
I think it would be ok to make TZ to which we convert to configurable. That way dates would stay consistent and one could set time zone to their own.

It would require adding a new config for example `writerOpts.timeZone` and changing following lines that are doing time conversion:
https://github.com/conventional-changelog/conventional-changelog/blob/0e05028f70bbd3109e1a4b16262a9450153060de/packages/conventional-changelog-writer/index.js#L20

https://github.com/conventional-changelog/conventional-changelog/blob/0e05028f70bbd3109e1a4b16262a9450153060de/packages/conventional-changelog-writer/index.js#L64

@medikoo if you believe that would be good for you and you are willing to invest some of your time,   I highly encourage you to make a pull-request with that feature.

**Comment by medikoo** [comment]
> By default time is converted to UTC. If we would use local timezone, it would be possible for two people to make two consecutive releases with first annotated with date that comes after the second release and that would confuse people even more.

I agree it's a safe approach. Still situation where project releases are done by different people in very different timezone is quite rare, and current behavior is unexpected for all teams working far from UTC timezone (mostly developers in New Zealand and Australia).

Assuming my above statement is true, more user friendly would be to have behavior reversed. So by default date from local timezone is used, and there's an option to rely on UTC (if one works in a project where versions are issued from very different parts of a globe)

**Comment by stevemao** [comment]
You can supply `date` in the `context` object:

https://github.com/conventional-changelog/conventional-changelog/tree/master/packages/conventional-changelog-writer#date

**Comment by TheLouTenant** [comment]
@stevemao I'm using standard-version and didn't manage to make the date configuration work with the convention-changelog-writer. Usually I use only the CLI to change the version and create the changelog. It is possible to make it work or I would need to configure everything from scratch?

**Comment by dangreen** [comment]
Dates are normalized to UTC by default to keep releases consistent across time zones. Applications that need another representation can override formatDate or provide context.date. No default change is planned. Closing as not planned.
